Assets/Scripts/Timers/Timer.cs

```diff
@@ -38,6 +38,10 @@ public class Timer : MonoBehaviour
     /// Runs once when the timer stops.
     /// </summary>
```

Comment:
Is there any need to Serialize the timer components if they are all controlled by a different script?